include/internal/libspdm_common_lib.h

```diff
@@ -265,6 +265,37 @@ typedef struct {
 #define spdm_context_struct_VERSION 0x1
 
 typedef struct {
+    //
+    // reserved buffer for request in the main dispatch function
+    // in SPDM responder. This buffer is the transport message
+    // recived from spdm_context->receive_message(). This message may
+    // be SPDM transport message or secured SPDM transport message.
+    // This buffer may be shared by last_spdm_request.
```

Comment:
I think sharing with last_spdm_request may be problematic.

In SPDM 1.2, request message is transport layer message, while last_spdm_request is SPDM message, which might be larger than transport layer message if CHUNK is supported.

Can we copy it to app_message ?